packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -359,6 +368,21 @@ class AppBar extends StatefulWidget implements PreferredSizeWidget {
   /// Defaults to [NavigationToolbar.kMiddleSpacing].
   final double titleSpacing;
 
+  /// The amount of height to be used for the [title].
+  ///
+  /// If not provided, then defaults to [kToolbarHeight].
+  final double titleHeight;
+
+  /// Specifies whether the [AppBar.leading] and [AppBar.actions] should always be placed vertically center of the AppBar.
```

Comment:
nit: doc strings should be wrapped at 80 chars.

not nit: I think you mean horizontally, right?

also not nit: why are we adding this property?  I think we should avoid adding it until we know someone wants it.